src/extension/content-script/batteries/GitHub.ts

```diff
@@ -3,75 +3,115 @@ import setLightningData from "../setLightningData";
 
 const urlMatcher = /^https:\/\/github.com\/([^/]+)(\/([^/]+))?$/;
 
-const battery = (): void => {
+const battery = async (): Promise<void> => {
   const urlParts = document.location.pathname.split("/");
   const username = urlParts[1];
   const repo = urlParts[2];
 
+  let found = false;
```

Comment:
`found` what? Could this be named more expressive?

Reply:
Sure.
I've refactored this entire code to be cleaner, let me know if you like this more.

Reply:
Cool thanks!